src/modules/hmis/hmisUtil.ts

```diff
@@ -181,6 +181,9 @@ export const parseAndFormatDate = (
   dateString: string | null | undefined
 ): string | null => {
   if (!dateString) return null;
+  // Trim string to date only
+  if (dateString.length > 10) dateString = dateString.slice(0, 10);
```

Comment:
this is fine, probably don't need the condition though?
```suggestion
  // remove time from ISO8601 date-time string
  dateString = dateString.slice(0, 10);
```